packages/bpk-animate-height/src/AnimateHeight.js

```diff
@@ -123,6 +123,9 @@ class AnimateHeight extends Component {
   onTransitionEnd() {
     if (this.contentElement && this.props.height === 0) {
       this.contentElement.style.display = 'none';
+      if (this.props.onAnimationComplete !== null) {
```

Comment:
so this is technically valid because we're using react's `PropTypes.func` , but a better check would be to use `typeof`, you'll invoke `onAnimationComplete` so check if it's a function and invoke it